src/addons/turbo.cpp

```diff
@@ -8,6 +8,9 @@
 
 #include <algorithm>
 
+#define TURBO_ENABLED 0
```

Comment:
You'll want to make sure these are not in here (turbo_enabled and turbo_led_pin) otherwise they are ALWAYS defined as these numbers. The #ifndef clause makes these come from boardconfig.h